kolibri/plugins/coach/assets/src/app.js

```diff
@@ -71,6 +71,9 @@ class CoachToolsModule extends KolibriApp {
         PageNames.LESSON_EDIT_DETAILS_BETTER,
         PageNames.LESSON_PREVIEW_SELECTED_RESOURCES,
         PageNames.LESSON_PREVIEW_RESOURCE,
+        PageNames.LESSON_SELECT_RESOURCES_INDEX,
+        PageNames.LESSON_SELECT_RESOURCES_BOOKMARKS,
+        PageNames.LESSON_SELECT_RESOURCES_CHANNELS,
```

Comment:
Well, we don't have to do the version that I think, since we seem to think the opposite thing is what makes the most sense! I'm not necessarily correct. Maybe `INDEX` could continue to be the landing page (which does make sense) and then `CHANNELS` could become something like `TOPIC_TREE`?

Reply:
Makes sense to me. Updated! :)